lib/experimental/fonts-library/class-wp-rest-fonts-library-controller.php

```diff
@@ -0,0 +1,216 @@
+<?php
+/**
+ * Rest Fonts Library Controller.
+ *
+ * This file contains the class for the REST API Fonts Library Controller.
+ *
+ * @package    Gutenberg
+ * @subpackage Fonts Library
+ * @since      X.X.X
+ */
+
```

Comment:
Add a `class_exists()` guard to avoid loading this class into memory after it is introduced into WordPress Core. See comment here https://github.com/WordPress/gutenberg/pull/52704/files#r1269629926.